server/lib/src/constants/acp.rs

```diff
@@ -1737,20 +1743,23 @@ lazy_static! {
             Attribute::AccountValidFrom,
             Attribute::PassKeys,
             Attribute::AttestedPasskeys,
+            Attribute::ApplicationPassword,
```

Comment:
Should these still be here? Per above it's the same logic that this role is for admins to reset other admins who forgot a password rather than needing to change their application passwords.

Reply:
Not pushed yet.